pyfod/quadrature.py

```diff
@@ -189,13 +189,14 @@ def _rs_weights(cls, grid, alpha=0.0):
 # ---------------------
 class GaussLegendreRiemannSum(object):
 
-    def __init__(self, NGLeg=5, NRS=20, pGLeg=0.9,
+    def __init__(self, NGLegDom=5, GlegDeg=4, NRS=20, pGLeg=0.9,
```

Comment:
To be consistent with previous naming conventions, I believe it should be "GLegDeg=4" in line 192.